apps/pie-docs/src/patterns/overlay-patterns/guidance/guidance.md

```diff
@@ -0,0 +1,386 @@
+            "Use them for succinct messaging.",
+            "They are triggered by the system."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use them to display long form content.",
+            "Don’t use them for non-essential information."
+        ]
+    }
+} %}
+
+---
+
+## Bottom sheet
+
+A bottom sheet is a surface attached to the bottom of the screen that displays additional information or actions while keeping the main content visible.
+
+{% notification {
+  type: "information",
+  iconName: "info-circle",
+  message: "Check out our [full documentation](https://pie.design/components/bottom-sheet/) for this component."
+} %}
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/overlay-patterns/bottom-sheet.svg",
+  alt: "A bottom sheet containing a heading and a list of action items.",
+  width: "360px"
+} %}
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use to display semi-critical information.",
+            "Use where the user needs to perform one specific task.",
+            "Use them as contextual interactions.",
+            "They are triggered by the user."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use them for multi-step flows. Use a modal view instead."
+        ]
+    }
+} %}
+
+---
+
+## Modal
+
+A modal is a surface that overlays the page’s main content and is used to display information, gather input or confirm actions.
+
+{% notification {
+  type: "information",
+  iconName: "info-circle",
+  message: "Check out our [full documentation](https://pie.design/components/modal/) for this component."
+} %}
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/overlay-patterns/modal.svg",
+  mobileSrc:"../../../../assets/img/patterns/overlay-patterns/modal-narrow.svg",
+  alt: "A modal containing a heading, multiple lines of text ant two buttons placed below the modal heading.",
+  width: "600px"
+} %}
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use them to display important information that needs user’s action or dismissal.",
+            "Use them when the user needs to perform one specific task.",
+            "They are triggered by the user."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use them for multi-step flows."
+        ]
+    }
+} %}
+
+---
+
+## Full page modal
+
+Also known as Modal View (iOS) or Scaffold View (Android) are a surface that overlays the page’s main content entirely and is used to display information, gather input or confirm actions.
+
+{% notification {
+  type: "information",
+  iconName: "info-circle",
+  message: "Check out our [full documentation](https://pie.design/components/modal/) for this component."
+} %}
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/overlay-patterns/frame.svg",
```

Comment:
I had a chat with Sian about this and in her opinion this image doesn't really match what she thinks of as a "full page modal"